src/transformers/convert_blenderbot.py

```diff
@@ -0,0 +1,114 @@
+# coding=utf-8
```

Comment:
The file should be named `convert_blenderbot_original_pytorch_checkpoint_to_pytorch.py`